src/platform/silabs/efr32/wifi/ethernetif.cpp

```diff
@@ -367,22 +404,37 @@ static err_t low_level_output(struct netif * netif, struct pbuf * p)
     /* Generate the packet */
     for (q = p, framelength = 0; q != NULL; q = q->next)
     {
+#if (SIWX_917 | EXP_BOARD)
+        wfx_rsi_pkt_add_data(packet, (uint8_t *) (q->payload), (uint16_t) q->len, framelength);
+#else
         wfx_rsi_pkt_add_data(rsipkt, (uint8_t *) (q->payload), (uint16_t) q->len, framelength);
+#endif
         framelength += q->len;
     }
     if (framelength < LWIP_FRAME_ALIGNMENT)
     {
         /* Add junk data to the end for frame alignment if framelength is less than 60 */
+#if (SIWX_917 | EXP_BOARD)
+        wfx_rsi_pkt_add_data(packet, (uint8_t *) (p->payload), LWIP_FRAME_ALIGNMENT - framelength, framelength);
```

Comment:
Seems like the same prototype, Why the ifdef? Can't you just rename `rsipkt` to `packet`